packages/react/src/components/Loading/Loading-test.js

```diff
@@ -7,12 +7,47 @@
 
 import React from 'react';
 import Loading from '../Loading';
-import { mount } from 'enzyme';
+import { mount, shallow } from 'enzyme';
 import { settings } from 'carbon-components';
+import { render, cleanup } from '@carbon/test-utils/react';
 
 const { prefix } = settings;
 
 describe('Loading', () => {
+  afterEach(cleanup);
+
+  describe('Automated Accessibility Testing', () => {
+    it('should have no Axe violations', async () => {
+      const { container } = render(<Loading />);
+      await expect(container).toHaveNoAxeViolations();
+    });
+
+    it('should have no DAP violations', async () => {
+      const { container } = render(<Loading />);
+      await expect(container).toHaveNoDAPViolations('Loading');
+    });
+  });
+
+  describe('Screenreader Accessibility', () => {
+    let wrapper;
+    let getLoader;
+
+    beforeEach(() => {
+      wrapper = shallow(<Loading id="test-id" />);
+      getLoader = () => wrapper.find('#test-id');
```

Comment:
Any way we could query by some other aria like `aria-live`?

Reply:
👍